tools/flang1/utils/symtab/symini_ftn.n

```diff
@@ -1297,6 +1297,8 @@ CM Fortran dotproduct intrinsic.
 .AT inquiry
 .PD PROCESSORS_SHAPE - -
 .AT inquiry
+.PD ISNAN - -
+.AT elemental x
```

Comment:
The following code shows link errors. Is there no support for the elemental version?
program isnan01
  real :: i(5)
  i = 0.0
  print *, isnan(i)
end program

Reply:
This is currently not supported.

Reply:
Can you remove the elemental description if elemental is not supported?

Reply:
I have added the support for elemental version.